### PR TITLE
Revert MSBuild to match VS 17.5.0 GA

### PR DESCRIPTION
**Pull request by rainersigwald**
**Description**

Automated merges of 17.5 builds done since the final 17.5.0 GA (but with no meaningful content) moved MSBuild in the SDK ahead of the one shipped with VS, which is not necessary or meaningful. Revert to match what's in VS ([internal link](https://dev.azure.com/devdiv/DevDiv/_git/VS?path=/.corext/Configs/components.json&version=GBrel/d17.5&line=39&lineEnd=39&lineStartColumn=19&lineEndColumn=83&lineStyle=plain&_a=contents)).

The reverted commits are: 
https://github.com/dotnet/msbuild/compare/6f08c67f3ed838dccfbcdab91e6bebc54a26fd94...81ac93548ee1044f03d0ac363712bb778920ef8c

**Customer Impact**
None, engineering change to align the shipped version in VS and .NET SDK.

**Regression?**
yes

**Risk** 
very low

### CODE REVIEW

**Review by marcpopMSFT — APPROVED** [review]

**Review by AR-May — APPROVED** [review]

**Comment by vlada-shubina** [comment]
@marcpopMSFT please consider disabling the subscription for release/7.0.2xx (at least temporary), otherwise correct version might be again overwritten.

There were PRs merged last week: https://github.com/dotnet/sdk/pulls?q=is%3Apr+release%2F7.0.2xx+Update+dependencies+from+dotnet%2Fmsbuild+ that supersedes correct version.

**Comment by rainersigwald** [comment]
Tactics discussion: approved under tell-mode if necessary, but please let me do some homework before merge.

**Comment by rainersigwald** [comment]
Ok, so 201 shipped with the same prerel MSBuild as 200

https://github.com/dotnet/sdk/blob/9bd5ecacbff29091ae2977870164608eaa576eed/eng/Version.Details.xml#L49-L52

So we should take this and 202 will be a strict forward move onto the official release bits.

**Pull request merged**
